cloudflare_worker/publish.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
```

Comment:
Consider dockerizing this.  Could be a TODO for now.

Reply:
Created cl/195126851